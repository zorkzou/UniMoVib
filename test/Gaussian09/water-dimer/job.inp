a test job

 $contrl
   qcprog="gaussian"
   ifgsva=.true.
 $end

 $gsva
   subsystem="1,2,3" 
 $end

 $qcdata
   fchk="0000-w2-lm0.fchk"
 $end
