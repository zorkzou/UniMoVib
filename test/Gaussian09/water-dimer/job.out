 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.3.0, Apr 29, 2020
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      Gaussian
 FCHK file:       0000-w2-lm0.fchk


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    O       8            1.36536579   -0.00006677    0.10009311           15.99491464
     2    H       1            1.77443064    0.76458040   -0.30815905            1.00782504
     3    H       1            1.77489034   -0.76400449   -0.30902361            1.00782504
     4    O       8           -1.50598688    0.00007608   -0.11934206           15.99491464
     5    H       1           -1.87611566   -0.00046588    0.76291170            1.00782504
     6    H       1           -0.54823652   -0.00018447    0.00826254            1.00782504
 ------------------------------------------------------------------------------------------


 <<< POINT GROUP SYMMETRY >>>
 Electronic Wavefunctions      :  C1  
 Nuclear & Total Wavefunctions :  C1  


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm^-1), reduced masses (AMU), force constants (mDyn/A), IR intensities (km/mol)

                                                   1                                 2                                 3
          Irreps                                   A                                 A                                 A
     Frequencies                            154.0102                          171.1304                          180.3057
  Reduced masses                              1.0620                            1.3080                            1.0428
 Force constants                              0.0148                            0.0226                            0.0200
  IR intensities                            183.3149                          195.2276                           16.9342
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8        -0.00002   0.00678  -0.00003       0.05442   0.00000  -0.03594       0.00012   0.04266   0.00005
           2   1        -0.16059  -0.12120  -0.40155       0.44575   0.00346   0.36552       0.56756  -0.23863   0.04379
           3   1         0.16114  -0.12157   0.40188       0.44576  -0.00356   0.36530      -0.56744  -0.23892  -0.04432
           4   8        -0.00001   0.05975   0.00005      -0.12549  -0.00004   0.00502      -0.00011  -0.02260  -0.00002
           5   1         0.00002  -0.76837  -0.00044       0.30592   0.00039   0.18671      -0.00016  -0.25431  -0.00018
           6   1        -0.00001  -0.04486  -0.00020      -0.06951   0.00024  -0.42688      -0.00003   0.41349   0.00029

                                                   4                                 5                                 6
          Irreps                                   A                                 A                                 A
     Frequencies                            205.1647                          382.0325                          685.7030
  Reduced masses                              3.1765                            1.1112                            1.0458
 Force constants                              0.0788                            0.0956                            0.2897
  IR intensities                            104.6603                          112.1584                          188.6235
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.29718  -0.00007   0.05291      -0.03042   0.00003  -0.06995       0.00000  -0.04117  -0.00002
           2   1        -0.09015  -0.00810  -0.35204       0.30440   0.00759   0.28629      -0.27191   0.09057  -0.02844
           3   1        -0.08868   0.00886  -0.35183       0.30426  -0.00774   0.28614       0.27181   0.09072   0.02850
           4   8        -0.23062   0.00006  -0.01999       0.02937  -0.00003   0.01478      -0.00001  -0.02897  -0.00002
           5   1        -0.59774   0.00007  -0.17338      -0.55073   0.00030  -0.23121       0.00006   0.02020   0.00004
           6   1        -0.27968  -0.00072   0.35478      -0.04135  -0.00021   0.53426       0.00017   0.91169   0.00058

                                                   7                                 8                                 9
          Irreps                                   A                                 A                                 A
     Frequencies                           1621.3290                         1638.5436                         3779.2729
  Reduced masses                              1.0843                            1.0764                            1.0552
 Force constants                              1.6794                            1.7028                            8.8800
  IR intensities                             91.9288                           36.3680                          351.4142
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.04498   0.00004  -0.04388       0.02661   0.00002  -0.01952       0.00305   0.00000  -0.00297
           2   1        -0.35153   0.37376   0.34327      -0.20693   0.20662   0.17558      -0.02456  -0.04355   0.02318
           3   1        -0.35130  -0.37436   0.34284      -0.20678  -0.20694   0.17535      -0.02459   0.04351   0.02323
           4   8        -0.01656   0.00002  -0.02969       0.02687  -0.00004   0.05260       0.05221  -0.00002   0.02048
           5   1         0.29699  -0.00014   0.12548      -0.52695   0.00025  -0.22229       0.12769   0.00013  -0.22925
           6   1        -0.04521  -0.00023   0.35596       0.09181   0.00044  -0.65363      -0.95543   0.00024  -0.09511

                                                  10                                11                                12
          Irreps                                   A                                 A                                 A
     Frequencies                           3894.5154                         3975.8469                         3999.1577
  Reduced masses                              1.0449                            1.0728                            1.0829
 Force constants                              9.3373                            9.9914                           10.2037
  IR intensities                             19.4751                           82.8280                           91.8684
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8        -0.03518  -0.00003   0.03490       0.00106   0.00000  -0.00101       0.00002  -0.07076  -0.00004
           2   1         0.27998   0.58313  -0.27725      -0.00768  -0.01802   0.00791       0.30217   0.56227  -0.30008
           3   1         0.28034  -0.58266  -0.27792      -0.00769   0.01801   0.00793      -0.30250   0.56174   0.30072
           4   8         0.00277  -0.00000   0.00302      -0.04029  -0.00003   0.05206       0.00000  -0.00001  -0.00000
           5   1         0.02158   0.00003  -0.04341       0.39399   0.00054  -0.88145      -0.00000   0.00007   0.00001
           6   1        -0.06745   0.00001  -0.00326       0.24393  -0.00008   0.05549      -0.00001  -0.00092  -0.00000

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                   A                                 A                                 A
     Frequencies                              0.0289                            0.0303                           -0.0362
  Reduced masses                              6.0035                            6.0035                            6.0035
 Force constants                              0.0000                            0.0000                           -0.0000
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           2   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           3   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           4   8         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           5   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           6   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825

                                                   4                                 5                                 6
          Irreps                                   A                                 A                                 A
     Frequencies                             47.4897                            7.3857                            6.1699
  Reduced masses                              1.0619                            5.1219                            5.0585
 Force constants                              0.0014                            0.0002                            0.0001
  IR intensities                              1.3573                            5.4245                            5.3792
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.00000  -0.03960  -0.00003      -0.02683   0.00027   0.35914      -0.00004  -0.35738   0.00026
           2   1        -0.01745   0.24943   0.52384       0.07802   0.00034   0.46435       0.19526  -0.45835   0.00685
           3   1         0.01745   0.25003  -0.52340       0.07854   0.00034   0.46448      -0.19500  -0.45846  -0.00616
           4   8         0.00000   0.04518   0.00003       0.02961  -0.00028  -0.37938       0.00004   0.37756  -0.00028
           5   1        -0.00000  -0.56770  -0.00035      -0.19731  -0.00034  -0.47458      -0.00026   0.46455  -0.00035
           6   1         0.00000  -0.02037  -0.00014      -0.00322  -0.00010  -0.13304      -0.00005   0.13196  -0.00010


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Molecular mass            :        36.021129    AMU
 Electronic total energy   :         0.000000    Hartree
 Scaling factor of Freq.   :         1.000000
 Tolerance of scaling      :         0.000000    cm^-1
 Rotational symmetry number:         1
 The C1   point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                  8.070504          276.487475          276.826317
           X                         0.999445           -0.000046            0.033303
           Y                        -0.000022           -1.000000           -0.000742
           Z                         0.033303            0.000741           -0.999445

 Rotational temperatures            10.732145            0.313265            0.312881    Kelvin
 Rot. constants A, B, C              7.459223            0.217730            0.217464    cm^-1
                                   223.621875            6.527389            6.519399    GHz


 #   1    Temperature =       298.15000 Kelvin         Pressure =         1.00000 Atm
 ====================================================================================

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.047128           29.573568
 Thermal correction to Energy               :            0.052801           33.133276
 Thermal correction to Enthalpy             :            0.053745           33.725761
 Thermal correction to Gibbs Free Energy    :            0.021175           13.287388
 ====================================================================================


 -- Generalized Subsystem Vibrational Analysis (GSVA) --

 Subsystem has     3  atoms:
    1    2    3


 *****************************************************************
 ***  Properties of Intrinsic Fragmental Vibrations from GSVA  ***
 *****************************************************************

 Results of vibrations:
 Normal frequencies (cm^-1), reduced masses (AMU), force constants (mDyn/A)

                                                   1                                 2                                 3
     Frequencies                           1606.5159                         3891.4598                         3997.1900
  Reduced masses                              1.0830                            1.0449                            1.0831
 Force constants                              1.6468                            9.3229                           10.1961
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.05011   0.00004  -0.05003      -0.03519  -0.00003   0.03514       0.00002  -0.07088  -0.00004
           2   1        -0.39774   0.42601   0.39726       0.27910   0.58591  -0.27853       0.30102   0.56269  -0.30042
           3   1        -0.39748  -0.42670   0.39677       0.27945  -0.58542  -0.27919      -0.30136   0.56217   0.30106


 UniMoVib job terminated correctly, Tue May 19 21:51:06 2020
